internal/store/postgres/asset_repository.go

```diff
@@ -633,31 +636,31 @@ func (r *AssetRepository) getOwners(ctx context.Context, assetID string) (owners
 }
 
 // insertOwners inserts relation of asset id and user id
-func (r *AssetRepository) insertOwners(ctx context.Context, execer sqlx.ExecerContext, assetID string, owners []user.User) (err error) {
+func (r *AssetRepository) insertOwners(ctx context.Context, execer sqlx.ExecerContext, assetID string, owners []user.User) error {
 	if len(owners) == 0 {
-		return
+		return nil
 	}
 
 	if !isValidUUID(assetID) {
 		return asset.InvalidError{AssetID: assetID}
 	}
 
-	var values []string
-	var args = []interface{}{assetID}
-	for i, owner := range owners {
-		values = append(values, fmt.Sprintf("($1, $%d)", i+2))
-		args = append(args, owner.ID)
+	sqlb := sq.Insert("asset_owners").
+		Columns("asset_id", "user_id")
+	for _, o := range owners {
+		sqlb = sqlb.Values(assetID, o.ID)
```

Comment:
Will this generate multiple `values` statements separated by commas?

Reply:
Yes, values statements with multiple comma seperated rows to be inserted in the table will be generated.
So in case the [] owners has two entries
```
VALUES (assetID, owners[0].ID), (assetID, owners[1].ID)
```